app-mobilephone/gammu/gammu-1.39.0.ebuild

```diff
@@ -35,10 +35,6 @@ RDEPEND="
 	dev-util/dialog
 	virtual/libiconv
 "
-PATCHES=(
-	"${FILESDIR}/${P}-smsd.patch"
-)
-
 src_configure() {
 	local mycmakeargs=(
```

Comment:
please change src_test not to muck with `MAKEOPTS` and instead use `LD_LIBRARY_PATH="${BUILD_DIR}/libgammu" cmake-utils_src_test -j1`